minimap.impl.user.js

```diff
@@ -418,22 +440,109 @@ const { html, render } = mlp_uhtml;
       rPlaceTemplate.botUrl !== undefined && settings.getSetting("bot").enabled
         ? rPlaceTemplate.botUrl
         : rPlaceTemplate.canvasUrl;
-    setTimeout(restoreBotWorkingRightNow, 10 * 1000);
```

Comment:
This failsafe restore should be kept, unless you know that we are going to always restore it.